bot.go

```diff
@@ -124,6 +124,17 @@ func (b *Bot) MessageReceived(channel *ChannelData, message *Message, sender *Us
 
 // SendMessage queues a message for a target recipient, optionally from a particular sender.
 func (b *Bot) SendMessage(target string, message string, sender *User) {
+	newMessage, err := b.executeFilterCommands(&FilterCmd{
+		Target: target,
+		Message: message,
+		User: sender})
+
+	if err == nil {
+		message = newMessage
+	}
+	if message == "" {
+		return
+	}
```

Comment:
I think it's better if the `executeFilterCommands` not return errors, see my other comment., this way you could simplify this code:

```Go
message = b.executeFilterCommands(....
```

As the executeFilterCommands will not change the message if the filter fails.